app/actions/action_pins.js

```diff
@@ -1,37 +1,40 @@
 import { DROP_NEW_PIN } from '../constants/constants.js';
-import {store } from 'redux';
+import { store } from 'redux';
 let nextPinId = 0;
 
-
-// const fire = new Firebase("https://interruptedlobster.firebaseio.com/user1");
-
 function dropNewPin(pinInfo) {
   return {
     type: DROP_NEW_PIN,
     id: nextPinId++,
-    long: pinInfo.long,
-    lat: pinInfo.lat,
+    longitude: pinInfo.longitude,
+    latitude: pinInfo.latitude,
     title: pinInfo.title
```

Comment:
@asu91 @jake-shasteen @bignell Either way works, not a huge difference. Since we don't have a way to save title right now, I think we should just take it out. And later have something pop up for the user to name upon clicking and/or have another function that can edit title whenever you click the marker. We can, however, use the spread operator in the current version for brevity.